pkg/cloudprovider/providers/gce/gce_test.go

```diff
@@ -437,3 +494,65 @@ func TestGenerateCloudConfigs(t *testing.T) {
 		}
 	}
 }
+
+func TestNewAlphaFeatureGate(t *testing.T) {
+	supportedAlphaFeatures["foo"] = true
```

Comment:
If you monkey patch the function, you need to store the old values and restore them at the end of the test. Otherwise, the subsequent tests will be affected.